src/slices/sidebar/discussion/comments-list/Node.js

```diff
@@ -36,6 +39,7 @@ export default class Node extends PureComponent {
 
     state = {
         editing: false,
+        enteredEditingMode: false,
```

Comment:
Perhaps @PedroMiguelSS could clarify what this is. We perhaps have discussed this in the past but I don't remember why it was necessary.

Reply:
Yes, the behavior is intended to be like this. This variable will be set to true on the first time the component has entered on editing mode. `editing` is the state property that manages the editing mode. But the main goal of `enteredEditingMode` is to control if editing mode was `true` at least once in the past.
This variable is needed to sync some of the animations and it's being used to avoid some classes from being applied again as you can see here:
`[styles.default]: !enteredEditingMode`

About the naming of this prop... well, be my guest to change it to something better! I admit it can be confusing 😄

Reply:
We have changed to `editingCompletedOnce`